client/src/components/modal-forms/ChangeSiteForm.js

```diff
@@ -42,7 +42,7 @@ export const ChangeSiteForm = ({ initialValues, sites, validationSchema, onSubmi
         <FormikForm>
           <Box my={1}>
             <FastField
-              name='startDate'
+              name='startDateAtNewSite'
```

Comment:
Is it okay to change this field, is there existing data that this might conflict with?